python/cudf_polars/cudf_polars/experimental/join.py

```diff
@@ -276,14 +277,17 @@ def _(
         partition_info[right].partitioned_on == ir.right_on
         and partition_info[right].count == output_count
     )
+    evaluate_with_tracing = do_evaluate_with_tracing(
+        ir.do_evaluate, name=type(ir).__name__
+    )
```

Comment:
(If you go with my suggested change to `do_evaluate_with_tracing`)

```suggestion
    evaluate_with_tracing = do_evaluate_with_tracing(ir)
```